regression-test/suites/rollup_p0/test_materialized_view_lazy_open.groovy

```diff
@@ -70,6 +75,58 @@ suite("test_materialized_view_lazy_open", "rollup") {
                 "dynamic_partition.end" = "3",
                 "dynamic_partition.prefix" = "p",
                 "dynamic_partition.buckets" = "32",
+                "light_schema_change" = "false",
+                "replication_num"="1"
+            );
+        """
+    
+    sql "DROP TABLE IF EXISTS ${tbName3}"
+    sql """
+            CREATE TABLE IF NOT EXISTS ${tbName4}(
+                k1 DATE,
+                k2 DECIMAL(10, 2),
+                k3 CHAR(10),
+                k4 INT NOT NULL
+            )
+            DUPLICATE KEY(k1, k2)
+            PARTITION BY RANGE(k1)
+            (
+               PARTITION p1 VALUES LESS THAN ("2000-01-01"),
+               PARTITION p2 VALUES LESS THAN ("2010-01-01"),
+               PARTITION p3 VALUES LESS THAN ("2020-01-01")
+            )
+            DISTRIBUTED BY HASH(k1) BUCKETS 32 
+
+            properties(
+                "light_schema_change" = "true",
```

Comment:
Even though we enable schema change here, we have not triggered schema change yet.